hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeSetOwnerRequest.java

```diff
@@ -153,6 +153,9 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
 
       // Set owner with new owner name.
       omVolumeArgs.setOwnerName(newOwner);
+      // Update modificationTime.
+      omVolumeArgs.updateModificationTime();
```

Comment:
Modification time should be set in the preExecute, and use the same value in validateAndUpdateCache. why it needs to be done like this is, in HA all OM's will have the same modification time, in this case there is a chance of OM's having different modification time.

Reply:
Thanks @bharatviswa504 for the info.
This info is important for me.
Updated.